docs/api/spec.md

```diff
@@ -184,28 +195,67 @@ try {
 catch (err) { ... }
 ```
 
-Raise an intent and retrieve data from the IntentResolution:
+Raise an intent and retrieve either data or a channel from the IntentResolution:
 ```js
 let resolution = await agent.raiseIntent("intentName", context);
 try {
-    const result = await resolution.getResult();
-    console.log(`${resolution.source} returned ${JSON.stringify(result)}`);
+  const result = await resolution.getResult();
+  if (result && result.broadcast) { //detect whether the result is Context or a Channel
+    console.log(`${resolution.source} returned a channel with id ${result.id}`);
+  } else if (){
+    console.log(`${resolution.source} returned data: ${JSON.stringify(result)}`);
+  } else {
+    console.error(`${resolution.source} didn't return anything`
+  }
 } catch(error) {
-    console.error(`${resolution.source} returned a data error: ${error}`);
+  console.error(`${resolution.source} returned a data error: ${error}`);
 }
 ```
 
 #### Resolvers
-Intents functionality is dependent on resolver functionality to map the intent to a specific App.  This will often require end-user input.  Resolution can either be performed by the Desktop Agent (for example, by displaying a resolver UI allowing the user to pick the desired app or app instance for the intent) or by the app handling the resolution itself (by using the `findIntents` API and specifying a target app or app instance when invoking the Intent), e.g.:
+Successful delivery of an intent depends first upon the Desktop Agent's ability to "resolve the intent" (i.e. map the intent to a specific App instance). Desktop Agents may resolve intents by any methodology. A common methodology is to display a UI that allows the user to pick the desired App for a given intent. Alternatively, the intent issuing app may proactively handle resolution by calling [`findIntent`](ref/DesktopAgent#findintent) or [`findIntentByContext`](ref/DesktopAgent#findintentbycontext) and then raising the Intent with a specific target application, e.g.:
```

Comment:
Should the last 'Intent' be capitalized?

Reply:
nope - well spotted